library/src/main/java/com/github/clans/fab/FloatingActionMenu.java

```diff
@@ -1152,15 +1156,19 @@ public Drawable getCloseIcon() {
     }
 
     // Netscout addition
-    public void setOpenIcon(Drawable icon) {
+    public void setOpenIcon(Drawable icon, int resId) {
         mOpenIcon = icon;
+        mOpenIconResId = resId;
         mImageOpenButton.setImageDrawable(mOpenIcon);
+        mImageOpenButton.setId(mOpenIconResId);
     }
 
     // Netscout addition
-    public void setCloseIcon(Drawable icon) {
+    public void setCloseIcon(Drawable icon, int resId) {
         mCloseIcon = icon;
+        mCloseIconResId = resId;
         mImageCloseButton.setImageDrawable(mCloseIcon);
+        mImageCloseButton.setId(mCloseIconResId);
```

Comment:
And here.

Reply:
No code changes for a location that actually calls this